cgd_tx_eff/src/edu/ohsu/compbio/annovar/annovar_parser.py

```diff
@@ -177,7 +179,8 @@ def _unpack_vf_transcript_tuple(self, delimited_transcript: str):
             # looks like "NM_001077690.1(NM_001077690.1:exon1:c.60+1C>-,NM_001077690.1:exon2:c.61-1C>-)"
             refseq_transcript, exon, hgvs_c_dot = self._parse_transcript_tuple_type5(delimited_transcript)
         else:
-            raise ValueError("Tuple format not recognized: " + str(delimited_transcript))
+            self.logger.warning("Annovar tuple format not recognized: " + str(delimited_transcript))
+            refseq_transcript, exon, hgvs_c_dot = None, None, None
```

Comment:
I wondered what were the implications of returning None here, but it looks like the only calling site in this review just skips None rows with this message:
`Ignoring transcript. Probably because it was intergenic and didn't have any useful tfx.`

Reply:
Yeah, a bit of processing does continue after those values are returned and I could probably just check for that right away and ``continue;`` but the end result is the same.